docker/images/android/stravenky/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Example build commands:
```

Comment:
As we discussed - I would prefer to keep project-specific configurations in project repos if possible.

Reply:
It´s possible to have a Dockerfile per project but this would require building a new docker image for every job. Support for this kind of use-case is not unified on all platforms e.g. GitlabCI and may require Docker-in-docker solution. So it doesn´t seem like a viable option.

Reply:
It should be possible, as we discussed in person. However, I'm still seeing those project-specific folders in the latest commit?